src/main/kotlin/graphics/scenery/ui/SwingBridgeFrame.kt

```diff
@@ -0,0 +1,191 @@
+package graphics.scenery.ui
+
+
+import graphics.scenery.utils.Image
+import graphics.scenery.utils.LazyLogger
+import java.awt.Component
+import java.awt.event.*
+import java.awt.image.BufferedImage
+import java.io.File
+import java.nio.ByteBuffer
+import javax.imageio.ImageIO
+import javax.swing.JFrame
+import javax.swing.JPanel
+import javax.swing.SwingUtilities
+
+/**
+ * @author Jan Tiemann <j.tiemann@hzdr.de>
+ * @author Konrad Michel <Konrad.Michel@mailbox.tu-dresden.de>
+ *
+ * Bridge to link the Swing window events with scenery key inputs in combination with SwingUiNode
+ */
+open class SwingBridgeFrame(title: String) : JFrame(title) {
+
+    val uiNode = SwingUiNode(this)
+
+    private var dragged = false
+    private var lastUpdate = 0L
+    private var snapshotBuffer : ByteBuffer? = null
+    var finalImage : Image? = null
```

Comment:
this is not a fitting name for this as a property. Maybe currentImage, currentSnapshot or just snapshot are more fitting
the terminology should be the same with the "snapshotBuffer"